rholang/src/test/scala/coop/rchain/rholang/interpreter/SubstituteTest.scala

```diff
@@ -0,0 +1,164 @@
+package coop.rchain.rholang.interpreter
+
+import org.scalatest.{FlatSpec, Matchers}
+import Substitute._
+import scala.collection.immutable.HashMap
+import SubstituteTest._
+
+object SubstituteTest {
```

Comment:
I don't like this at all. All of these are short enough that the tests would be more readable if this object were gone and you used the values directly. Tests are a specification and splitting these out makes them harder to read.